docs/partials/component-list.html

```diff
@@ -1,8 +1,8 @@
 <ul class="vertical menu docs-nav" id="docs-menu">
-  <p class="docs-nav-version">
+  <li class="docs-nav-version">
     <span data-docs-version></span>
-    <a href="https://github.com/zurb/foundation-sites/releases/" target="_blank">(Changelog)</a>
-  </p>
+    <a href="https://github.com/zurb/foundation-sites/releases/" target="_blank" style="display:inline-block;padding:0;">(Changelog)</a>
```

Comment:
Never use `styles="..."`, especially not to remove unwanted properties.

But I don't see clean solutions. It is properties of `docs-nav` affecting all `li > a` elements, inherited from `@mixin menu` ([here](https://github.com/zurb/foundation-sites/blob/1edf028dd4fcf27ef5c953388c3c20d034ef94df/scss/components/_menu.scss#L60)).

It is a classic `<tag>` selector problem, but fixing it would be a major breaking change.

Poke @kball : a perfect exemple of a no-BEM approach concrete problem :wink:. One more.